bin/bootstrap-cloud

```diff
@@ -34,6 +51,8 @@ check_aws_prerequisites() {
   return 0
 }
 
+check_terraform_prerequisites
```

Comment:
We don't need to check terraform for the k3d path, since bootstrap/stage0 doesn't use terraform here. Run this within the individual cloud matches, as necessary.